db/migrate/20150810234807_add_token_to_invitations.rb

```diff
@@ -0,0 +1,9 @@
+class AddTokenToInvitations < ActiveRecord::Migration
+  def change
+    add_column :invitations, :invitation_token, :string
+    Invitation.all.each do |invitation|
```

Comment:
It is not fully recommended to work with data migrations in place of schema migrations. The model name can change in future and totally wipe your from-start-to-end migrations. I would recommend creating custom rake task to handle this one time migration. - read more [here](http://guides.rubyonrails.org/command_line.html#custom-rake-tasks)